src_pop/core/biogeochem/POP.F90

```diff
@@ -6,11 +6,20 @@
 
 !CITATION
 !--------------------------------------------------------
-!When referring to this code in publications, please cite:
```

Comment:
I would start with a lead-in comment such as 

!> This MODULE contains the code for the POP model.  The principal components of the MODULE are the 
! initialisation SUBROUTINE [[InitPOP2D_Poission]], the time stepping SUBROUTINE [[POPStep]] and routines for 
! allometry, recruitment and disturbance.

then the existing inline documentation is ok

Reply:
Thank you for the Review, Ian. I have followed your suggestions. See committed code.